src/style.css

```
@@ -0,0 +1,60 @@
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 50px;
+  padding-left: 50px;
+}
+
+.header-logo {
+  height: 40px;
+  width: 40px;
+  margin-top: 10px;
+}
+
+.nav-menu {
+  height: 60px;
+  display: flex;
+}
+
+.nav-link {
+  text-decoration: none;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 60px;
+  color: #000;
+  margin: 0 10px;
+}
+
+.first-link {
+  margin-left: 0;
+}
+
+.last-link {
+  margin-right: 0;
+}
+
+.nav-link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  display: block;
+  position: relative;
+  bottom: 4px;
+  content: "";
+  width: 36px;
```

```suggestion
  width: 100%;
```
так ширина буде правильною для всіх лінок ,а не тільки для цього яке має 36 пікселів